sql/src/test/java/io/crate/analyze/expressions/ExpressionAnalyzerTest.java

```diff
@@ -161,6 +162,27 @@ public void testInSelfJoinCaseFunctionsThatLookTheSameMustNotReuseFunctionAlloca
     }
 
     @Test
+    public void testSwapFunctionLeftSide() throws Exception {
+        TableInfo tableInfo = mock(TableInfo.class);
+        when(tableInfo.getReferenceInfo(new ColumnIdent("x"))).thenReturn(
+                new ReferenceInfo(new ReferenceIdent(new TableIdent("doc", "t"), "x"), RowGranularity.DOC, DataTypes.INTEGER));
+        TableRelation tr1 = new TableRelation(tableInfo);
+
+        Map<QualifiedName, AnalyzedRelation> sources = ImmutableMap.<QualifiedName, AnalyzedRelation>of(
+                new QualifiedName("t"), tr1
+        );
+        ExpressionAnalyzer expressionAnalyzer = new ExpressionAnalyzer(
+                analysisMetaData, emptyParameterContext, new FullQualifedNameFieldProvider(sources), null);
+
+        Function cmp = (Function)expressionAnalyzer.convert(
+                SqlParser.createExpression("8 + 5 > t.x"), context
```

Comment:
Would it be possible to use `SqlExpressions.asSymbol("8 + 5 > x")` and remove the whole setup code here?